Decide on GI on colored graphs fixes #12

mvandermade opened the pull request.

mvandermade committed:
Loop over i and j to get all pairs

JKleinRot commented:
Dit lijkt goed, alleen zit er in het andere merge request van Vera ook al zoiets. Misschien even kijken hoe dat straks goed samen kan gaan. Ik vraag me alleen af hoe dit issue #12 oplost, want nu loop je over de verschillende graven, maar ik zie nog niet waar er dan wordt gezegd dat het isomorf is of niet.

mvandermade committed:
Merge branch 'master' into framework_color_refinement_decide_gi

mvandermade committed:
Disjoint union check and vertex labelling added to graph.py and graph.io

disjoint union and vertex labelling extension.

mvandermade committed:
Update main.py

Peform isomorphic checks.
Added the hints from the canvas.utwente.nl to the comments of filename
